src/utils/sortPeople.ts

```diff
@@ -0,0 +1,37 @@
+import { Person } from '../types/Person';
+import { SortBy } from '../types/SortBy';
+
+type SortArgs = {
+  filteredPeople: Person[],
+  sortBy: keyof SortBy | null,
+  order: string | null,
+};
+
+type SortPeopleType = (args: SortArgs) => Person[];
+
+export const sortPeople: SortPeopleType = (args) => {
+  const { filteredPeople, sortBy, order } = args;
+
+  if (sortBy) {
+    filteredPeople.sort((a: Person, b: Person) => {
```

Comment:
`sort` and `reverse` are methods that mutate an initial array, so don't forget to make a copy before calling them